tests/Doctrine/Tests/ORM/Functional/EntityRepositoryTest.php

```diff
@@ -724,24 +725,27 @@ public function testFindByAssociationArray(): void
         $repo = $this->_em->getRepository(CmsAddress::class);
         $repo->findBy(['user' => [1, 2, 3]]);
 
+        $loggedQuery        = $this->getLastLoggedQuery();
+        $loggedQuery['sql'] = strtolower($loggedQuery['sql']);
+
         if (! class_exists(LoggingMiddleware::class)) {
             // DBAL 2 logs queries before resolving parameter positions
             self::assertSame(
                 [
-                    'sql' => 'SELECT t0.id AS id_1, t0.country AS country_2, t0.zip AS zip_3, t0.city AS city_4, t0.user_id AS user_id_5 FROM cms_addresses t0 WHERE t0.user_id IN (?)',
+                    'sql' => strtolower('SELECT t0.id AS id_1, t0.country AS country_2, t0.zip AS zip_3, t0.city AS city_4, t0.user_id AS user_id_5 FROM cms_addresses t0 WHERE t0.user_id IN (?)'),
```

Comment:
How come a functional test makes assertions against SQL? It's pretty pointless, I'd remove it instead.

Reply:
Unfortunately, I did not write these tests. I can't say why they decided to add them. I just brought it to mind for comparison, to. Oracle all converts to uppercase unless explicitly specified.

Reply:
@greg0ire these are more or less the same queries that you're working on in https://github.com/doctrine/orm/pull/9890. Please consider the removal of the tests that cause issues instead of wasting time on maintaining them.